src/components/HighlightedName/index.tsx

```diff
@@ -0,0 +1,71 @@
+import ColorHash from 'color-hash';
+import assign from 'lodash/assign';
+import memoize from 'lodash/memoize';
+import * as React from 'react';
+import styled from 'styled-components';
+import asRendition from '../../asRendition';
+import { Coloring, RenditionSystemProps, Theme } from '../../common-types';
+import { getColor, isLight } from '../../utils';
+import Txt, { TxtProps } from '../Txt';
+
+const colorHash = new ColorHash();
+const backgroundColor = memoize((text: string): string => {
+	return colorHash.hex(text.replace(/\s/g, ''));
+});
+
+const BaseHighlightedName = styled(Txt.span)`
+	font-family: ${props => props.theme.titleFont};
+	display: inline-block;
+	border-radius: 2px;
+	line-height: 1;
+	color: ${props => props.color};
+`;
+
+const HighlightedName = ({
+	children,
+	className,
+	theme,
+	...props
```

Comment:
You can also add the `color` here, and change `props.color` below to just `color`

Reply:
I agree, but it seemed like the convention was to do `props.foo` for most stuff rather than destructure so I went with that. I can (and would prefer to) destructure everything though.